features/error_formatting.feature

```diff
@@ -65,6 +65,7 @@ Feature: Error formatting
 
       1) Scenario: some scenario # features/a.feature:3
          ✔ Given a basic step # features/step_definitions/cucumber_steps.js:3
+            ℹ Basic info.
```

Comment:
Thoughts on this being: `Attachment (text/plain): Basic info.` 
Then other non text attachments could be `Attachment (other/type)`
Currently the figures are only used for steps and not related messages and I'd like to keep that pattern

Reply:
Ok, no figure. How should I stringify other media types ?

Reply:
For the other media types, I was thinking the value could be suppressed. So for now the line would be `Attachment (other/type)`. We can add json in another PR. Yes please add some docs to the attachment page.

Reply:
Ok